module.txt

```diff
@@ -5,18 +5,10 @@
     "displayName" : "SubstanceMatters",
     "description" : "Adds systems for tracking the composition of items.  Also adds world generation for some core ores if OreGeneration is enabled.",
     "dependencies" : [
-        {
-          "id": "Workstation", "minVersion": "1.0.0"
-        },
-        {
-            "id": "OreGeneration", "minVersion": "1.0.0", "optional": true
-        },
-        {
-            "id": "Fluid", "minVersion": "1.0.0"
-        },
-        {
-            "id": "ItemRendering", "minVersion": "1.0.0"
-        }
+        {"id": "Workstation", "minVersion": "1.0.0"},
```

Comment:
Should this explicitly depend on `CoreAssets` and `CoreBlocks`?

Reply:
I would say so, yeah. It works via indirect dependency in the JS tree right now, but the module explicitly refers to those to so yep we should add them in :-)